client/src/components/Home/styled.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+import { Theme } from "Theme/theme";
```

Comment:
You shouldn't need to import the theme like this. It'll be passed in as props with theme provider. Works well to decouple